project-template/templates/deployment.yaml

```diff
@@ -21,8 +21,8 @@ spec:
       annotations:
         {{- include "project-template.annotations" . | nindent 8 }}
     spec:
-      serviceAccountName: {{ include "project-template.namespace" . }}-user
-      automountServiceAccountToken: false
+      #serviceAccountName: {{ include "project-template.namespace" . }}-user
+      #automountServiceAccountToken: false
```

Comment:
Wir sollten hier weiterhin den `serviceAccoutName` und `automountServiceAccountToken` setzen. Kriegen wir das trotz NetworkPolicy irgendwie hin?

Reply:
@JulianKahnert: Die habe ich lokal bei mir auskommentiert, da ich sonst nicht mit meinem Admin-Account arbeiten könnte. Ich schreibe ein ToDo in den Pull-Request, dass ich das beim finalen push wieder einkommentiere.